### DISCUSSION TITLE
EnvKey v1 broken since update yesterday

### DISCUSSION THREAD

**Discussion by sbleon** [envkey/envkey#9] [Q&A] [answered]
Hello. My EnvKey v1 updated yesterday and now it's broken with this JavaScript error:
![Screen Shot 2022-04-28 at 1 46 26 PM](https://user-images.githubusercontent.com/198711/165955249-4baf3bb5-d1a8-4bbb-b1a2-daddff064dcb.png)

I would downgrade, but I can't find a place to download any version of EnvKey v1. Is there a place I can download v1?

**Comment by sbleon** [comment, marked as answer]
Nevermind. I found it! https://github.com/envkey/envkey-app/releases

1.4.26 is working for me.

**Reply by danenania** [reply]
Sorry about that @sbleon! Glad you found the working release quickly.
